torch_xla/csrc/runtime/pjrt_computation_client.cc

```diff
@@ -896,7 +925,11 @@ PjRtComputationClient::GetReplicationDevices() {
 }
 
 xla::PjRtDevice* PjRtComputationClient::StringToPjRtDevice(
-    const std::string& device) {
+    const std::string& raw_device) {
+  std::string device = raw_device;
+  if (raw_device=="AOT:0") {
```

Comment:
question: is this a new flag or a recycle of an existing flag in the upstream torch?

Reply:
`AOT` is the newly added device type, similar to `SPMD` device flag. Both only exist in torch_xla.